test/vaadin-combo-box-light.html

```diff
@@ -181,6 +197,28 @@
         ).to.equal('foo');
       });
     });
+    
+    describe('nested template', () => {
+      let comboBox;
+
+      beforeEach(() => {
+        comboBox = fixture('combobox-nested-template');
+        comboBox.items = ['bar', 'baz', 'qux'];
+      });
+
+      it('should use the root level template for the item template', () => {
```

Comment:
nit: preferably, the test description should say what it actually checks:
```
it('should not use nested template as the item template', () => {
```

not a blocker, can be changed with `[skip ci]` in the commit message.

Reply:
Updated.